doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -420,6 +423,151 @@ an additional feature and still be conformant with Anuket.
 [here](../../../ref_model/chapters/chapter05.md#5.1) and hardware profile
 features [here](../../../ref_model/chapters/chapter05.md#5.4).
 
+
+<a name="3.2.2.1"></a>
+ ### 3.2.2.1 Kubernetes Networking Semantics
+The support for traditional network orchestration is non existing in Kubernetes proper. Kubernetes is foremost a Platform as a Service (PaaS) environment and not an Infrastructure as a Service (Iaas) infrastructure component. There is no orchestration API like Neutron in Openstack and there is no way to create L2 networks, instantiate network services such as L3aaS and LBaaS and then a way connect them all together like in case of Neutron.
```

Comment:
```suggestion
The support for advanced network configuration management is non existing in Kubernetes proper. Kubernetes is foremost a Platform as a Service (PaaS) environment and not an Infrastructure as a Service (Iaas) infrastructure component. There is no orchestration API like Neutron in Openstack and there is no way to create L2 networks, instantiate network services such as L3aaS and LBaaS and then a way connect them all together like in case of Neutron.
```